tests/backup/backup_basic_test.go

```diff
@@ -159,7 +160,7 @@ var _ = BeforeSuite(func() {
 			CreateBucket(provider, globalAzureBucketName)
 			log.Infof("Bucket created with name - %s", globalAzureBucketName)
 		case drivers.ProviderGke:
-			globalGCPBucketName = fmt.Sprintf("%s-%s", globalGCPBucketPrefix, bucketNameSuffix)
+			globalGCPBucketName = fmt.Sprintf("%s", globalGCPBucketPrefix)
```

Comment:
Why was this changed?

Reply:
fixed the code if the bucket name is empty it was generating the name with -- which is invalid